st/clitests/policy_files/iam-policy.json

```diff
@@ -8,7 +7,7 @@
         "s3:PutBucketAcl"
       ],
       "Effect": "Allow",
-      "Resource": "arn:aws:s3:::samplebucket"
+      "Resource": "arn:aws:iam::samplebucket"
```

Comment:
Wrong change.
PutBucketAcl is S3 service so previous arn is correct.
plz change it original

Reply:
done